src/operations/blas1_trees.hpp

```diff
@@ -445,10 +445,11 @@ PORTBLAS_INLINE bool TupleOp<rhs_t>::valid_thread(
     cl::sycl::nd_item<1> ndItem) const {
   return ((ndItem.get_global_id(0) < get_size()));
 }
+
 template <typename rhs_t>
 PORTBLAS_INLINE typename TupleOp<rhs_t>::value_t TupleOp<rhs_t>::eval(
     typename TupleOp<rhs_t>::index_t i) {
-  return TupleOp<rhs_t>::value_t(i, cl::sycl::fabs(rhs_.eval(i)));
```

Comment:
How does this relate to removing ComputeCpp?

Reply:
With the removal of ComputeCpp, I also checked that all exprtest work on DPC++. The `collapse_nested_tuple` test was disabled for it due to an llvm compiler error, but this is not true anymore. The test failed for DPC++ for this `sycl::fabs` which is not necessary anymore.
To give you more context, I think this function was here for the previous implementation of `iamax` and `iamin` that used to return the absolute value and the index. I thought the `abs` was still necessary for `iamax` and `iamin` but current operators implemented inside the kernel already compute the absolute value for each number, so there is no need to do it also here. So now all exprtest and unittest pass with DPC++.